### PR TITLE
fix tabs tsx

### PR DESCRIPTION
**Pull request by bccsafe**
* static member `TabPane`
* properties `pageSize`, `speed`, `tabBarhammerOptions`

#667 

First of all, thanks for your contribution! :-)

Please makes sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [x] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

**Comment by bccsafe** [comment]
@paranoidjk ci上和本地`lint`规则不同？

File: components/tabs/index.web.tsx [hunk 1]
```diff
@@ -6,13 +6,10 @@ import TabContent from 'rc-tabs/lib/TabContent';
 import InkTabBar from 'rc-tabs/lib/InkTabBar';
 import SwipeableInkTabBar from 'rc-tabs/lib/SwipeableInkTabBar';
 
-const Tabs = React.createClass<TabsProps, any>({
-  statics: {
-    TabPane,
-  },
+export default class Tabss extends React.Component<TabsProps, any> {
```
Comment (paranoidjk) [inline components/tabs/index.web.tsx]: ~~`Tabss`~~ --> `Tabs`

File: components/tabs/index.web.tsx [hunk 2]
```diff
@@ -6,13 +6,10 @@ import TabContent from 'rc-tabs/lib/TabContent';
...
+  static TabPane = TabPane;
```
Comment (paranoidjk) [inline components/tabs/index.web.tsx]: 旧的 `statics: {}` 这个语法是啥,好像没见过，是因为这样才导致 TabPane 有定义错误吗

Comment (bccsafe) [inline components/tabs/index.web.tsx]: 对的 应该是老的写法 typescript不认，生成的declare有问题

**Pull request merged**